docs/ABOUT.md

```diff
@@ -0,0 +1,46 @@
+[Dart](https://en.wikipedia.org/wiki/Dart_(programming_language)) is a general-purpose computer programming language that is:
+ 1. [Concurrent](https://en.wikipedia.org/wiki/Concurrent_computing)
+ 2. [Class-Based](https://en.wikipedia.org/wiki/Class-based_programming)
+ 3. [Object-Oriented](https://en.wikipedia.org/wiki/Object-oriented_programming)
```

Comment:
What is the most exciting thing to you about Dart? Or what is your favorite thing about it? Or what drew you to it in the first place? I'd like the opening sentence or two to be something that speaks to why people might be excited about it rather than a factual bullet-point list.

I don't think we need to refer to wikipedia when we mention that Dart is concurrent, class-based, and object-oriented. People can search for the terms if they need to.